commands/root.go

```diff
@@ -1,21 +1,21 @@
 package commands
 
 import (
-	"github.com/bndw/pick/config"
+	c "github.com/bndw/pick/config"
 	"github.com/spf13/cobra"
 )
 
 // Global pick config
-var Config *config.Config
+var config *c.Config
 
 var rootCmd = &cobra.Command{
 	Use:   "pick",
 	Short: "pick is a minimal password manager",
 	Long:  "pick is a minimal password manager",
 }
 
-func Execute(cfg *config.Config) error {
-	Config = cfg
+func Execute(cfg *c.Config) error {
+	config = cfg
```

Comment:
Maybe name the global `safeConfig`. Well, I guess we need to stick with c, cfg and config.